src/common/TimeInterval.js

```diff
@@ -69,4 +72,4 @@ class TimeInterval {
 	}
 }
 
-module.exports = TimeInterval;
+module.exports = { TimeInterval };
```

Comment:
We should be able to default export the TimeInterval class, e.g. `module.exports = TimeInterval`.

Reply:
You already asked me about this in person, and already I told you that it doesn't work properly. I can't figure out a way to get it to work like that, but if you can I'd love to get this back to a default export.